src/android/CDVMobilePayAppSwitch.java

```diff
@@ -209,7 +209,7 @@ public void onFailure(FailureResult result) {
                     }
                 }
                 @Override
-                public void onCancel() {
+                public void onCancel(String string) {
```

Comment:
Was this supposed to go into the error object?

Reply:
No, turns out this is actually the order ID. I've removed the (seemingly) temporary variable called `_inflightOrderId` since it was only used for `onCancel`.

Reply:
Ah cool, the inflight stuff was because you couldn't track multiple payments iirc, so had to hack around that back then

Reply:
I thought so, nice that we have this now :)